src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -181,6 +181,7 @@ internal static class LogMessages
         public const string IDX10685 = "IDX10685: Unable to Sign, Internal SignFunction is not available.";
         public const string IDX10686 = "IDX10686: Unable to Verify, Internal VerifyFunction is not available.";
         public const string IDX10687 = "IDX10687: Unable to create a AsymmetricAdapter. For NET45 or NET451 only types: '{0}' or '{1}' are supported. RSA is of type: '{2}'..";
+        public const string IDX10688 = "IDX10688: RSACryptoServiceProviderProxy creation is only supported on Windows platform.";
```

Comment:
>platform [](start = 118, length = 8)

pls. remove this (platform) word. (otherwise the sentence needs to end with ".. the Windows platform."